CharlieBackend.Api/Startup.cs

```diff
@@ -147,6 +148,8 @@ public void ConfigureServices(IServiceCollection services)
             services.Configure<SwaggerOptions>(c => c.SerializeAsV2 = true);
 
             services.AddSwaggerExamplesFromAssemblyOf<Startup>();
+
+            services.AddTransient<IJWTGenerator, JWTGenerator>();
```

Comment:
Please move this line to the `CompositionRoot.cs`

Reply:
bc2aae1